packages/table/src/table.tsx

```diff
@@ -1030,48 +1081,31 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         const rowIndexEnd = showFrozenRowsOnly ? numFrozenRows : rowIndices.rowIndexEnd;
 
         return (
-            // <div
```

Comment:
Went ahead and deleted this unrelated commented code. Was a relic from the frozen rows/columns refactor.